roles/load-balancers/manage-haproxy/templates/lb_backend.j2

```diff
@@ -1,51 +1,49 @@
+{% set lb_ssl = namespace(enabled=False) -%}
+{% set lb_mode = namespace(http=False) -%}
 
 {% for entry in lb_config.lb_entries %}
 
 {% if ((entry.lb_mode is defined and entry.lb_mode == 'http') or
        (entry.lb_mode is undefined and entry.lb_match_port|string == '80')) %}
-{% set lb_http_mode = True %}
+{% set lb_mode.http = True %}
 {% else %}
-{% set lb_http_mode = False %}
-{% endif %}
+{% set lb_mode.http = False %}
```

Comment:
Can these lines be removed since the default is `lb_mode.http = False`?

Reply:
Need to "reset" on every iteration of the loop, so no.